packages/styling/src/styles/theme.ts

```diff
@@ -105,6 +108,19 @@ export function createTheme(theme: IPartialTheme, depComments: boolean = false):
     ...theme.semanticColors
   };
 
+  if (theme.typography) {
+    const { types } = theme.typography;
+    for (const typeName in types) {
+      if (typeName) {
+        const type = types[typeName] as IFontType;
```

Comment:
This casting makes me a bit nervous. These seem to be defined as `Partial<IFontType>`; by casting we are saying they are not Partial. Are we causing potential problems elsewhere?

Reply:
I can change the casting to `Partial<IFontType>`

Reply:
Actually I'm hoping you can remove the cast entirely... if I'm reading this correctly the type should already be `Partial<IFontType>`

Reply:
Right! That makes sense. I'll just remove it and add null checks for the values then.